tools/src/bin/teehistorian2demo.rs

```diff
@@ -91,13 +91,14 @@ fn process(in_: &Path, out: &Path) -> Result<(), String> {
     let mut snap_buffer = Vec::new();
     let mut th;
     let mut demo;
+    let mut outt = std::io::Cursor::new(Vec::new());
```

Comment:
The buffer must probably be declared before the writer. I guess that's the lifetime issue.

Reply:
Nice catch, I'm not used to those declarations ^^